range-finder.js

```diff
@@ -0,0 +1,48 @@
+function sort(array) {
+  return array.sort(function(a, b) {
+    return a - b;
+  });
+}
+
+function findRanges(sortedInput) {
+  let minContinous = 0;
+  let maxContinous;
+  let continousCount = 1;
+  const ranges = [];
+  sortedInput.forEach((element, index) => {
+    const difference = sortedInput[index + 1] - sortedInput[index];
```

Comment:
Will sortedInput[index+1] work fine for the last index? will not cause out of range exception or index out of bounds?

Reply:
sortedInput[index + 1] has undefined value. 
Undefined - Number will give NaN which is handled in line no 14.
JS will not throw out of range exception or index out of bounds.

Reply:
Then your design is relying on the exception. It is always better to have a clear design than relying on the existing way. Of course the code works now, but it is not guaranteed to be same in future, if some garbage integer value is returned there is no way of finding the fault.

Reply:
updated